src/providers/Wallet.js

```diff
@@ -40,8 +40,9 @@ function WalletAugmented({ children }) {
 
 function WalletProvider({ children }) {
   const chainId = getDefaultChain()
+  const network = getNetwork(chainId)
 
-  const connectors = getUseWalletConnectors()
+  const connectors = getUseWalletConnectors(network)
```

Comment:
wondering if we can do something similar to swapr, because in my opinion doesn't make to much sense to me change a library that is not being maintained anymore for another library that neither is being maintaned, but if we can tune it in some way like the swapr one would be pretty cool, having the ability to choose the network and change between gardens deployments without the need of use another endpoint 

<img width="1319" alt="Screen Shot 2021-05-21 at 9 04 52 PM" src="https://user-images.githubusercontent.com/11763623/119208367-49af5000-ba78-11eb-91e5-63ce95462f05.png">

<img width="390" alt="Screen Shot 2021-05-21 at 9 07 44 PM" src="https://user-images.githubusercontent.com/11763623/119208457-a14dbb80-ba78-11eb-926b-6d4bee6e9a71.png">

Reply:
Yo can discard this comment